plugins/crypto/main/src/azkaban/crypto/CryptoV1_1.java

```diff
@@ -0,0 +1,66 @@
+package azkaban.crypto;
+
+import java.security.Provider;
+
+import org.apache.log4j.Logger;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.jasypt.encryption.pbe.PBEStringEncryptor;
+import org.jasypt.encryption.pbe.StandardPBEStringEncryptor;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.base.Preconditions;
+
+/**
+ * Uses AES algorithm to encrypt and decrypt.
+ */
+public class CryptoV1_1 implements ICrypto {
+  private static final Logger logger = Logger.getLogger(CryptoV1_1.class);
+
+  private static final String CIPHERED_TEXT_KEY = "val";
+  private static final String CRYPTO_ALGO = "PBEWITHSHA256AND128BITAES-CBC-BC";
+  private static final Provider PROVIDER = new BouncyCastleProvider();
+
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  @Override
+  public String encrypt(String plaintext, String passphrase, Version cryptoVersion) {
+    Preconditions.checkArgument(Version.V1_1.equals(cryptoVersion));
```

Comment:
That's right. I thought it's quite small amount of code and worth not sharing. Let me know if you prefer sharing the code.